aries-site/src/pages/components/skeleton.mdx

```diff
@@ -39,3 +42,4 @@ The components that currently support skeleton are:
 - Text, Paragraph, Heading
 - Button
 - Box, Card, Pageheader, Footer, Header
+- DataSearch
```

Comment:
DataSearch component doesn't support skeleton out of the box with grommet at the moment, so let's remove this for now.
```suggestion
```